db/migrate/20210129021138_set_user_alt_email_to_null_and_add_unique.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class SetUserAltEmailToNullAndAddUnique < ActiveRecord::Migration[5.0]
+  def change
+    execute "UPDATE users SET alt_email=NULL where alt_email=''"
```

Comment:
how single quotes saved in alt_email column?

Reply:
In future again we will get same issue. We have to fix while saving  the alt_email.

Reply:
Sure.  Currently, we have a unique constraint on that column.  So there can be at most only one empty alt_email row for the table.  We also have code in the users.rb model file that makes sure alt_email is unique before the user object is persisted to the database.  
I've updated the users model to take allowed_blank out for that field just now, so that should finish covering the '' scenario.